test/unit/app/workers/test-calculate-workload-points.js

```diff
@@ -183,7 +183,7 @@ describe('services/workers/calculate-workload-points', function () {
 
     return calculateWorkloadPoints.execute(task).then(function () {
       expect(
-        updateWorkloadPointsCalculationsStub.calledWith(REPORT_ID, WORKLOAD_ID, WORKLOAD_ID, expectedTotalPoints, SDR_POINTS, SDR_POINTS,
+        updateWorkloadPointsCalculationsStub.calledWith(REPORT_ID, WORKLOAD_ID, WORKLOAD_ID, WORKLOAD_ID, expectedTotalPoints, SDR_POINTS, SDR_POINTS,
```

Comment:
I think it's worth explicitly calling out `WORKLOAD_POINTS_ID` and `T2A_WORKLOAD_POINTS_ID` as their own vars